obci/drivers/etr/etr_amplifier_ws_peer.py

```diff
@@ -8,61 +8,105 @@
 
 from obci.drivers.etr import etr_amplifier
 from obci.configs import settings, variables_pb2
+from multiplexer.multiplexer_constants import peers, types
 
 from ws4py.client.threadedclient import WebSocketClient
 
 class EtrWebSocketClient(WebSocketClient):
-    def __init__(self, peer):
-        super(EtrWebSocketClient, self).__init__()
+    def __init__(self, url, peer):
+        '''
+        :param url: Url to websocket wich provides data
+        :param peer: EtrWebSocketAmplifier peer'''
+        super(EtrWebSocketClient, self).__init__(url)
         self.peer = peer
 
     def opened(self):
-        print("Opened")
+        self.peer.logger.info("Opened")
 
     def closed(self, code, reason):
-        print(("Closed down", code, reason))
+        self.peer.logger.info(("Closed down", code, reason))
         self.peer.end_the_loop = True
 
     def received_message(self, m):
-        try:
-            msg = json.loads(m)
-            print("x = {}, y = {}".format(m['x'], m['y']))
-            
-            l_msg = variables_pb2.Sample2D()
-            l_msg.x = float(m['x'])
-            l_msg.y = float(m['y'])
-            l_msg.timestamp = float(m['timestamp'])
-            
-            self.peer.process_message(l_msg)
-        except Exception as ex:
-            pass
-
+    
+        msg = json.loads(str(m))
+        self.peer.logger.info("msg: {}".format(m))
+        l_msg = variables_pb2.Sample2D()
+        l_msg.x = float(msg['x'])
+        l_msg.y = float(msg['y'])
+        l_msg.timestamp = float(msg['timestamp'])
+        
+        self.peer.process_message(l_msg)
 
-class EtrWebSocketAmplifier(etr_amplifier.EtrAmplifier):
-    def __init__(self):
-        super(EtrWebSocketAmplifier, self).__init__()
+class EtrWebSocketClientSample(EtrWebSocketClient):
+    '''
+    Modification to EtrWebSocketClient, which lets to capture more data
+    than x and y gaze position.
+    '''
+    def __init__(self, url, peer,  packet = 4):
+        super(EtrWebSocketClientSample, self).__init__(url, peer,)
         
+        self.packetsize = packet
+        self.msg_vector = variables_pb2.SampleVector()
+    def received_message(self, m):
+            
```

Comment:
like:
2016-04-12 17:09:59,587 - etr_amplifier - INFO - Opened
2016-04-12 17:09:59,609 - etr_amplifier - ERROR - 
            
            ERROR: UNAVAILABLE CHANNELS:
            ooo
            
            
            Channels from tracker server JSON:
            y, timestamp, eye_detected, x
            
            You've tried to access channels:
            
            timestamp, eye_detected, x, y, ooo
            
            
            
            
2016-04-12 17:09:59,609 - etr_amplifier - INFO - ('Closed down', 1006, 'Going away')